### DISCUSSION TITLE
Custom build flash 'the file version is error!'

### DISCUSSION THREAD

**Discussion by Dav3Plus** [egzumer/uv-k5-firmware-custom#163] [General]
Hello!

Thanks for your hard work to produce this!  I followed your clear instructions to create a user customization build, but on using the official  v1.1.12 update tool to flash, it states 'the file version is error!' .  I already have your 0.19 build installed!  Any ideas?

Regards... David.

**Comment by Dav3Plus** [comment]
...I worked it out (I'm just a novice😁).  The .bin firmware needs to be' packed' after reading the 'win_make.bat' file instructions.   Just curious, the original untouched make file config creates a compile error!
   "bi/bin/ld.exe: region `FLASH' overflowed by 96 bytes
  collect2.exe: error: ld returned 1 exit status"🤔

Regards... David.

**Reply by ErikS-web** [reply]
Did you see this answer #161

**Reply by Dav3Plus** [reply]
Thank you, yes, I read about using Docker as it's more efficient at packing!👍
